sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyAsyncClient.java

```diff
@@ -260,15 +279,7 @@ public Mono<Response<KeyVaultKey>> getKeyWithResponse() {
             }
         } else {
             return FluxUtil.monoError(LOGGER,
-                new UnsupportedOperationException("Operation not supported when in operating local-only mode"));
-        }
-    }
-
-    Mono<JsonWebKey> getSecretKey() {
-        try {
-            return implClient.getSecretKeyAsync();
-        } catch (RuntimeException ex) {
-            return monoError(LOGGER, ex);
+                new UnsupportedOperationException("Operation not supported when operating in local-only mode"));
```

Comment:
Should add to the Javadocs that this will fail is local cryptography is being used. And do we have a way for inspecting whether local cryptography is being used? If not, we probably should.

Reply:
Agreed, good catch.